index.html

```diff
@@ -308,20 +308,153 @@
 
     <section class="informative">
       <h2>Introduction</h2>
+
+      <p>
+Digital signatures, based on
+<a href="https://en.wikipedia.org/wiki/Public-key_cryptography">asymmetric
+cryptography</a>, can be used in [=authentication=] and [=authorization=]
+schemes to make them difficult for adversaries to compromise. However,
+one shortcoming of digital signatures is the challenge in disseminating
+necessary information, such as public cryptographic keys, to those who need
+to verify the security of a digital signature.
```

Comment:
```suggestion
to verify the security of a digital signature. Another potential shortcoming is notifying the holder of a public key when it should no longer be used.
```
Either this or the above, since the problem is not only disseminating the public keys. It is the whole management lifecycle of them.

Reply:
I don't understand how this is "another" "potential" shortcoming. I don't see any other (potential or definite) shortcomings discussed in the prior text. This suggestion needs more work before it should be considered ready to merge.

Reply:
I attempted to make the language simpler in 7a2f27db9168f5a4b6e37a51f7bd77a90a47b2d5.